src/redist/targets/GenerateBundledVersions.targets

```
@@ -480,7 +501,7 @@ Copyright (c) .NET Foundation. All rights reserved.
                       TargetFramework="net6.0"
                       AppHostPackNamePattern="Microsoft.NETCore.App.Host.**RID**"
                       AppHostPackVersion="$(_NET60RuntimePackVersion)"
-                      AppHostRuntimeIdentifiers="@(NetCoreAppHostRids, '%3B')"
+                      AppHostRuntimeIdentifiers="@(NetCore60AppHostRids, '%3B')"
```

```suggestion
                      AppHostRuntimeIdentifiers="@(Net60AppHostRids, '%3B')"
```

Theres also the `NetCoreRuntimePackRids` reference in a .NET 6.0 just above that could be changed to `Net60RuntimePackRids` https://github.com/dotnet/installer/blob/fe8ff26c2f24d1a92cfdf04a945dc820a818e7db/src/redist/targets/GenerateBundledVersions.targets#L497